multimodal/tests/unittests/others/test_object_detection.py

```diff
@@ -123,6 +124,12 @@ def test_mmdet_object_detection_inference_basics(checkpoint_name):
     # Inference on data without annotations
     pred = predictor.predict(test_path_with_images_only)
 
+    # Save inference in COCO on data without annotations
+    pred = predictor.predict(test_path_with_images_only, save_results=True)
+
+    # Save inference in COCO on data without annotations
+    pred = predictor.predict(test_path_with_images_only, save_results=True, as_coco=True)
+
```

Comment:
I can't really see a difference between the comments. 🤔

Reply:
Fixed.